src/index.html

```diff
@@ -1,18 +1,84 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&family=Space+Grotesk:wght@700&family=Space+Mono&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/style.css"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body
+    class="page__body"
+    data-qa="card"
+  >
+    <header class="header">
+      <div class="image">
+        <img
+          class="image_mac"
+          src="images/imac.jpeg"
+          alt="Imac image"
+        />
+      </div>
+
+      <div class="header__text">
+        <h1 class="header__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h1>
+
+        <h2 class="header__product">Product code: 195434</h2>
```

Comment:
Consider will be better if you use the tag p instead of h2, fix it everywher
```suggestion
        <p class="header__product">Product code: 195434</p>
```